.yarn/versions/6b81de03.yml

```diff
@@ -0,0 +1,14 @@
+releases:
+  "@yarnpkg/builder": patch
+  "@yarnpkg/esbuild-plugin-pnp": patch
+
+declined:
+  - "@yarnpkg/plugin-constraints"
+  - "@yarnpkg/plugin-exec"
+  - "@yarnpkg/plugin-interactive-tools"
+  - "@yarnpkg/plugin-stage"
+  - "@yarnpkg/plugin-typescript"
+  - "@yarnpkg/plugin-version"
+  - "@yarnpkg/plugin-workspace-tools"
+  - "@yarnpkg/cli"
+  - "@yarnpkg/pnp"
```

Comment:
Since `@yarnpkg/builder` is only a devDependency and the fix does not affect Yarn repo itself, it seems reasonable to skip releases for these packages.
Is that correct?

Reply:
Yes, that's correct. `@yarnpkg/builder` itself technically doesn't need a patch release since semver will pick this up on its own but it's fine.